phpstan-baseline.neon

```diff
@@ -1,26 +1,36 @@
 parameters:
 	ignoreErrors:
-		-
-			message: "#^Used function DDTrace\\\\root_span not found\\.$#"
-			path: src/Command/RunCommand.php
-			reportUnmatched: false
```

Comment:
to tu nech prosímtě

Reply:
prozradis mi proc? 🤔

Reply:
Protože ti to pak bude svítit v phpstormu pokud si to otevřeš lokálně a nemáš nainstalovanej datadog a phpstan si na tom nespustíš

Reply:
Vratil jsem ti to zpatky 
https://github.com/keboola/job-runner/pull/269/commits/92cd72e1cdb4e830b56c31fab56fc623a4f37ecb